custom_components/octopus_energy/services.yaml

```diff
@@ -289,4 +289,49 @@ boost_heat_pump_zone:
       selector:
         number:
           step: 0.5
+          mode: box
+
+set_heat_pump_flow_temp_config:
+  name: Set heat pump flow temperature configuration
+  description: Sets the fixed and/or weather compensated flow temperatures
```

Comment:
issue: Based on the fields, it looks like it's fixed and weather compensation not one or the other as all fields are required.